.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# Mood Analysis
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
+
+|  Question 	|  Answer 	|
+|---	|---	|
+|  Why does 03/13 come out as neutral when it should be happy? How could we fix this?	|   	|
+|  Where should we call strip_punctuation? Does it matter? Why?	|   	|
+|  What are you going to do if there aren't at least 3 happy days? Where do you need to handle that case? 	|   	|
+|  Should you use an array or a hash to solve this problem? Why? 	|   	|
+|  What if we eventually want to add feelings to our analysis? Can we write this code in a way that will prevent us from having to re-write it later? 	|   	|
```

Comment:
I'm not sure what this question is asking - is it possible to make it more specific?